### PR TITLE
GPP Extensions: DE, IA, NE, NH, NJ, TN State Sections

### PR DESCRIPTION
**Pull request by jaredmoscow**
### Summary
This pull request introduces updates for GPP extensions with new state sections for Delaware, Iowa, Nebraska, New Hampshire, New Jersey, and Tennessee. These additions are essential for accommodating evolving privacy regulations. There are also updates to the Section Information, reflecting the addition of these new States.

### Details

- **GPP Extensions & New State Sections:** This PR includes the creation of dedicated folders and README files for each of the mentioned states. The technical specifications for each state aim to aid with GPP support and help implementers stay ahead of regulatory requirements.
- **Public Comment Period:** It's important to note that this PR is open for public comment until September 5th, 2024. Feedback can be provided either by directly adding comments to this PR or by sending an email to support@iabtechlab.com.
- **Commit Overview:** The PR includes 16 commits for the new state sections. Commit messages provide insight into each change noted in this review process.

### Changes Overview

- **Delaware:** Creation of the DE State Section folder along with its README file and Delaware technical specification.
- **Iowa: Introduction of the IA State Section Folder and README.md for Iowa, along with the GPP extension based on Iowa’s privacy technical specification.
- **Nebraska:** Creation of the Nebraska folder and its respective README.md file, accompanied by the GPP extension for Nebraska’s technical specification.
- **New Hampshire:** Creation of the NH State Section folder, including its README.md, and the GPP extension adhering to New Hampshire’s technical specification.
- **New Jersey:** Creation of the NJ State Section folder, including its README.md, and the GPP extension adhering to New Jersey’s technical specification.
- **Tennessee:** Creation of the TN State Section folder, including its README.md, and the GPP extension adhering to Tennessee’s technical specification.
- **Section Information Update:** In addition to state-specific section updates, additions have been made to the Section Information.md file, including new rows for these six States to ensure coverage.

### Next Steps
Your feedback and suggestions are highly appreciated during this public comment period. We encourage thorough review and constructive criticism to refine the implementation further. Once public comment is complete and the feedback is incorporated, we will proceed with merging these changes into the main branch.

### CODE REVIEW

File: Sections/US-States/NJ/GPP Extension: IAB Privacy’s New Jersey Privacy Technical Specification.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+        <td>Two bits for each Data Activity:<p></p><code>0</code> = Not Applicable, the Controller does not Process the specific category of Sensitive Data<p></p><code>1</code> = No Consent<p></p><code>2</code> = Consent&nbsp;<span style="color:rgb(36, 41, 47);"><p></p>(1). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Racial or Ethnic Origin.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(2). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Religious Beliefs.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(3). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing a Mental or Physical Health Condition, Treatment, or Diagnosis.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(4). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Sex Life or Sexual Orientation.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(5). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Status as Transgender or Nonbinary.</span><p></p>
```
Comment (bretg) [inline Sections/US-States/NJ/GPP Extension: IAB Privacy’s New Jersey Privacy Technical Specification.md:140]: Please consider moving transgender to number 9, shifting 6-9 up one each. See the same comment for Delaware for background on why.

File: Sections/US-States/DE/GPP Extension: IAB Privacy’s Delaware Privacy Technical Specification.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        <td>Two bits for each Data Activity:<p></p><code>0</code> = Not Applicable, the Controller does not Process the specific category of Sensitive Data<p></p><code>1</code> = No Consent<p></p><code>2</code> = Consent&nbsp;<span style="color:rgb(36, 41, 47);"><p></p>(1). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Racial or Ethnic Origin.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(2). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Religious Beliefs.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(3). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing a Mental or Physical Health Condition or Diagnosis (Including Pregnancy).</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(4). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Sex Life or Sexual Orientation.</span><p></p>
+          <span style="color:rgb(36, 41, 47);">(5). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Status as Transgender or Nonbinary.</span><p></p>
```
[the remark this answers is not in the document]
Comment (bretg) [reply Sections/US-States/DE/GPP Extension: IAB Privacy’s Delaware Privacy Technical Specification.md:140]: Any feedback from the committee on this suggestion?

[the remark this answers is not in the document]
Comment (jaredmoscow) [reply Sections/US-States/DE/GPP Extension: IAB Privacy’s Delaware Privacy Technical Specification.md:140]: @bretg public comment ended last Thursday, 9/5. The Global Privacy Working Group will be addressing comments and working towards finalization in the next few meetings - our next scheduled meeting is this Thursday, 9/12. We'll confirm once this is addressed and what actions are taken. thanks!

**Comment by jaredmoscow** [comment]
@bretg updates have been included for NJ and DE with the commits on this thread

**Review by bretg — APPROVED** [review]
Excellent - thanks @jaredmoscow - that makes the mapping much cleaner.

**Review by lamrowena — APPROVED** [review]

File: Sections/US-States/IA/GPP Extension: IAB Privacy’s Iowa Privacy Technical Specification.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+        <td>
+          <span style="color:rgb(36, 41, 47);">MspaCoveredTransaction</span>
+        </td>
+        <td>Int(2)</td>
+        <td>
+          <span style="color:rgb(36, 41, 47);">Publisher or Advertiser, as applicable, is a signatory to the IAB Multi-State Privacy Agreement (MSPA), as may be amended from time to time, and declares that the transaction is a “Covered Transaction” as defined in the MSPA.&nbsp;</span><p></p><code>1</code> = Yes<p></p><code>2</code> = No</td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">MspaOptOutOptionMode</span>
```
Comment (bretg) [inline Sections/US-States/IA/GPP Extension: IAB Privacy’s Iowa Privacy Technical Specification.md:176]: @jaredmoscow - this file seems to be missing AdditionalDataProcessingConsent... was that intentional?

**Pull request merged**